bedrock/settings/static_media.py

```diff
@@ -1399,13 +1377,13 @@
             'js/mozorg/home/ga-tracking.js',
             'js/mozorg/home/scroll-prompt.js',
         ),
-        'output_filename': 'js/home-2015-bundle.js',
+        'output_filename': 'js/home-bundle.js',
     },
-    'home-2015-ie9': {
+    'home-ie9': {
```

Comment:
We have a `matchmedia_addlistener` bundle set up for these cases (where it's just the addListener shim needed). Better to use that bundle and remove this one.